src/main/java/com/example/JavaChallenge6/web/IndexController.java

```diff
@@ -0,0 +1,23 @@
+package com.example.JavaChallenge6.web;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+
+@Controller
+public class IndexController {
+
+    @GetMapping
+    public String index(Model model) {
+        LocalDateTime today = LocalDateTime.now();
+        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("yyy年MM月dd日 HH時mm分ss秒");
+        String dateTimeJa = dateTimeFormatter.format(today);
```

Comment:
このあたりは一行にしてしまっても良いかもです！

```suggestion
  String formattedDateTime = DateTimeFormatter.ofPattern("yyy年MM月dd日 HH時mm分ss秒").format(LocalDateTime.now());
        model.addAttribute("formattedDateTime", formattedDateTime);
```

Reply:
ご指摘ありがとうございます。
修正いたしましたのでご確認お願いいたします。
また、一行にまとめれることができることを知り、勉強になりました。
ありがとうございました。